pandas/tests/arrays/test_integer.py

```diff
@@ -829,6 +829,14 @@ def test_arrow_array(data):
     assert arr.equals(expected)
 
 
+@pytest.mark.parametrize("ddof", [0, 1])
```

Comment:
can you parameterize this over kurt, skew, sem as well

Reply:
Done